doc/source/whatsnew/v0.23.0.txt

```diff
@@ -295,8 +295,10 @@ Other Enhancements
 - ``IntervalIndex.astype`` now supports conversions between subtypes when passed an ``IntervalDtype`` (:issue:`19197`)
 - :class:`IntervalIndex` and its associated constructor methods (``from_arrays``, ``from_breaks``, ``from_tuples``) have gained a ``dtype`` parameter (:issue:`19262`)
 - Added :func:`SeriesGroupBy.is_monotonic_increasing` and :func:`SeriesGroupBy.is_monotonic_decreasing` (:issue:`17015`)
+- :func:``DataFrame.apply`` keeps the specified ``Series`` subclass when ``Series`` and ``DataFrame`` subclasses are defined (:issue:`19822`)
```

Comment:
I find this not so clear. It is about the Series type that is passed to the function?

Reply:
Yes.  
When `apply`ing a function to a subclassed `DataFrame` I realised I could not call the specific methods of the subclassed `Series` that this `DataFrame` was supossed to generate  through `_constructor_sliced`. As far as I understood, the subclassing of the `Series` was lost during `apply`.  
With these changes the subclass is now kept inside `apply`, so method of the expected `Series` subclass can be used.

Reply:
OK, can you try to clarify the text a bit? Eg something like "For subclassed DataFrames, DataFrame.apply will now preserve the Series subclass (if defined) when passing the data to the applied function" (but adapt as you like)

Reply:
sure! I like it as this! :)
will change it!